### DISCUSSION TITLE
Show abstract of Talk

### DISCUSSION THREAD

**Discussion by navlisilvan** [EventFahrplan/EventFahrplan#465] [Q&A] [answered]
Hi, we try Eventfahrplan with Pretalx and i am wondering if Eventfahrplan could show the abstract int the shedule. 
In the exported XML there is a "subtitle" tag but i don´t see where to add this in Pretalx :/

**Comment by johnjohndoe** [comment, marked as answer]
As far as I know Pretalx does not offer a `subtitle` tag. The tag is there in order to support Frab backends.

In order to show the `abstract` text as a `subtitle` in the main schedule view you could customize the app. Please keep in mind that there is little space to display the text. Therefore, authors should compose very short texts for the abstract/subtitle.

**Reply by navlisilvan** [reply]
how to customize the app?

In Pretalx you can configure the max. lenght of abstract, so i thik it would fit.

**Reply by navlisilvan** [reply]
so the data of this screenshot ist from Frab?

https://github.com/EventFahrplan/EventFahrplan/blob/master/gfx/EventFahrplan-36c3-Events-Tablet-Phone.png

**Reply by johnjohndoe** [reply]
The screenshot is from the app for the 36c3 event. There the schedule data comes from various backends which use different software (Frab, Pretalx).

With regards to customizing the handling of `abstract` resp. `subtitle` have a look at the `Session.sanitize()` function in [SessionExtensions.kt#L165](https://github.com/EventFahrplan/EventFahrplan/blob/master/app/src/main/java/nerd/tuxmobil/fahrplan/congress/dataconverters/SessionExtensions.kt#L165).

**Reply by navlisilvan** [reply]
thaks a lot! :)

**Reply by johnjohndoe** [reply]
Sounds as if it worked out.
